### PR TITLE
adding Stack search

### PR DESCRIPTION
**Pull request by ghost**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

Its a code in java how to search an item in stack

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by HarshAgarwal-DEV** [comment]
@iamrajiv @HarshCasper  please have a look at that it is an simple stack search telling how to search an element in stack

**Comment by ghost** [comment]
@iamrajiv @HarshCasper  please have a look at that it is an simple stack search telling how to search an element in stack

File: Java/search/Stack_Search.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import java.util.*;
```
Comment (atarax665) [inline Java/search/Stack_Search.java:3]: Add description of Stack search at the top.

File: Java/search/Stack_Search.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        }
+        // Displaying the Stack 
+        System.out.println("The stack is: " + st); 
+         //enter the item to be searched
+            int item=sc.nextInt();
+        // Checking for the item
+            int search=st.search(item);
+           System.out.println(item+" is seached at index="+(n-search+1));     
+    } 
+
```
Comment (atarax665) [inline Java/search/Stack_Search.java]: Refer to [Java style guide](https://google.github.io/styleguide/javaguide.html) to write more readable code.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
@atarax665  have a look again if correct then merge it hope so it is up to mark now

**Review by atarax665 — APPROVED** [review]
Follow [Java style guide](https://google.github.io/styleguide/javaguide.html) to improve the readability of the code.

**Comment by atarax665** [comment]
@HarshAgarwal-DEV You can't name the class 'Stack' as it will conflict with Stack.java, though I have rectified and updated the code.

**Comment by ghost** [comment]
@atarax665  though you have rectified and updated the code.
why its not merged yet or i have to do some changes

**Comment by atarax665** [comment]
LGTM! Good job @HarshAgarwal-DEV

**Pull request merged**